config/puma/development.rb

```diff
@@ -0,0 +1,56 @@
+# Puma can serve each request in a thread from an internal thread pool.
```

Comment:
Part of my trying to figure out how to get the better_error stuff working: posts online suggested that it didn't play nicely with multithreaded servers, since the was no guarantee the debugger would run in the same thread context as the busted code. All I changed I'm the development context was the number of threads, to 1.